README.md

```diff
@@ -91,7 +91,28 @@ For the `edit` role, in addition to the permissions available to `view`:
 
 Users authenticated against OpenShift OAuth will be added to the Jenkins authorization matrix upon their first successful login.
 
-Permissions for users in Jenkins can be changed in OpenShift after those users are initially established in Jenkins.  The OpenShift Login plugin polls the OpenShift API server for permissions and will update the permissions stored in
+Now, with v1.0.10 of this plugin, you can change
+
+* Which OpenShift Roles are checked for authorization
+* Which Jenkins permissions map to which OpenShift Roles
+
+This plugin will look at for a ConfigMap named openshift-jenkins-login-plugin-config.  Typically, when running Jenkins in an OpenShift Pod, it 
+will look in the namespace that Jenkins is running in.  Otherwise, it looks in the namespace specified in the "client ID" as explained in ["Secondary Scenarios" down below.](#secondary-scenarios) 
+
+If this plugin finds and can read in that ConfigMap, it then:
+
+* Checks the `data` map for a key named `roleList`
+* If found, the plugin expects it to be a comma separated list of OpenShift Roles to perform authorization review against.
```

Comment:
given that we know the list of roles they care about, from the roles they reference in their permission/role mapping entries, do we need this?  Can't we just construct the set of roles to SAR ourselves by looking at all the roles that are referenced in their mappings?

Reply:
agreed